src/status_im2/subs/general.cljs

```diff
@@ -198,6 +198,22 @@
  (fn [seed]
    (or seed {:step :intro})))
 
+(re-frame/reg-sub
+ :bottom-sheet/config
```

Comment:
two points here
1. (get-in db [:bottom-sheet/config should be a root sub
2. you could use destructuring with :or default values